src/System.Security.Principal.Windows/tests/System.Security.Principal.Windows.Tests.builds

```diff
@@ -4,7 +4,7 @@
   <ItemGroup>
     <Project Include="System.Security.Principal.Windows.Tests.csproj">
       <OSGroup>Windows_NT</OSGroup>
-      <TestTFMs>netcore50;netcoreapp1.0;net46;netcoreapp;net463</TestTFMs>
+      <TestTFMs>netcore50;netcoreapp1.0;net46;netcoreapp1.1;net463</TestTFMs>
```

Comment:
This file should actually be deleted now as we have deleted all ".builds" as we aren't using them after the engineering change.